packages/vx-tooltip/package.json

```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "@types/classnames": "^2.2.9",
     "@types/react": "*",
-    "@vx/bounds": "0.0.195",
+    "@vx/bounds": "^0.0.195",
```

Comment:
can you actually revert this? version bumps are managed by `lerna` upon release.